src/Jackett.Common/Definitions/exttorrents.yml

```
@@ -132,7 +127,7 @@ search:
       selector: td:nth-child(1) div a
       attribute: href
     download:
```

```suggestion
    infohash:
      selector: a[href^="magnet:?xt="]
      attribute: href
      filters:
        - name: regexp
          args: ([A-F|a-f|0-9]{40})
```
some clients do not add any default trackers so the magnet sits in limbo for ages waiting for a peer connect to supply the metadata.
I think it's better to use the infohash which does a better job building a more friendly magnet url with a title and trackers included.